backend/test/deal-observer.test.js

```diff
@@ -1,10 +1,10 @@
 import assert from 'node:assert'
 import { after, before, beforeEach, describe, it } from 'node:test'
 import { createPgPool, migrateWithPgClient } from '@filecoin-station/deal-observer-db'
-import { fetchDealWithHighestActivatedEpoch, storeActiveDeals } from '../lib/deal-observer.js'
 import { ActiveDealDbEntry } from '@filecoin-station/deal-observer-db/lib/types.js'
 import { Value } from '@sinclair/typebox/value'
 import { BlockEvent } from '../lib/rpc-service/data-types.js'
+import { fetchDealWithHighestActivatedEpoch, storeActiveDeals } from '@filecoin-station/deal-observer-db/lib/database-access.js'
```

Comment:
Is there any particular reason why you are importing data-access functions from a specific file inside the other module instead of exposing data-access functions in the public API of `@filecoin-station/deal-observer-db`?

I vaguely remember importing from a specific file local to a 3rd-party-package file can break under certain circumstances, but I don't remember the details. 

Personally, I prefer this:

```js
import { 
  createPgPool, 
  fetchDealWithHighestActivatedEpoch,
  storeActiveDeals, 
  migrateWithPgClient,
} from '@filecoin-station/deal-observer-db'
```

@juliangruber WDYT? Do you share my concern about importing from files in module's `lib` folder?